### PR TITLE
Test

### PR DESCRIPTION
**Pull request by DannyMcwaves**
new fixes made as per feature requests[WIP]
error in console in oneplayer mode fixed.

### CODE REVIEW

**Comment by DannyMcwaves** [comment]
screen shot 2018-08-28 at 9 13 24 pm

screen shot 2018-08-28 at 9 13 30 pm

**Comment by DannyMcwaves** [comment]
@JoshuaVSherman these tests pass locally.

**Comment by DannyMcwaves** [comment]
@JoshuaVSherman  do you get this linter errors when using node v10.9.0?

screen shot 2018-08-28 at 10 48 27 pm

**Comment by JoshuaVSherman** [comment]
I will handle this when we are back home

**Comment by JoshuaVSherman** [comment]
@DannyMcwaves please comment out this test and push up the code, thanks
FAILED TESTS:
  the App module
    ✖ gets the current styles with dashboard/reader route
      Chrome 68.0.3440 (Linux 0.0.0)
    TypeError: Cannot read property 'style' of undefined
        at <Jasmine>
        at App.setOtherStyles (webpack-internal:///app:5414:19)
        at App.get (webpack-internal:///app:5710:12)
        at UserContext.eval (webpack-internal:///./test/karma-unit/app.spec.js:335:19)
        at <Jasmine>

File: package-scripts.js [hunk 1]
```diff
@@ -29,9 +29,9 @@ module.exports = {
         fix: 'eslint --fix'
       },
       react: {
-        default: crossEnv('BABEL_TARGET=node jest --no-cache --config jest.React.json --notify'),
-        accept: crossEnv('BABEL_TARGET=node jest -u --no-cache --config jest.React.json --notify'),
-        watch: crossEnv('BABEL_TARGET=node jest --watch --no-cache --config jest.React.json --notify')
+        default: crossEnv('BABEL_TARGET=node jest --no-cache --config jest.React.json --notify --updateSnapshot'),
+        accept: crossEnv('BABEL_TARGET=node jest -u --no-cache --config jest.React.json --notify --updateSnapshot'),
```
Comment (JoshuaVSherman) [inline package-scripts.js]: @DannyMcwaves the update snapshot should only be 'accept' please fix this, thanks

**Pull request merged**